src/main/java/uk/gov/hmcts/reform/hmc/data/HearingEntity.java

```diff
@@ -110,6 +111,32 @@ public Optional<HearingResponseEntity> getLatestHearingResponse() {
             : Optional.empty();
     }
 
+    public String getHearingStatus() {
```

Comment:
Now it's in a common area as a `public` method, it'd be worth adding a couple of tests specifically for this method